docs/uab_cloud/security_setup_basic.md

```diff
@@ -0,0 +1,111 @@
+# Basic Security Setup
+
+Security Groups can and should be added as needed. 
```

Comment:
Can we add a little bit more to this intro? I'd like to maintain the intent of these docs to be a walkthrough for mostly-novice users. Something like the following:

```Using SSH with your instances requires following the steps on this page. At a minimum, an SSH security group and at least one keypair must be created. Other security groups can and should be added as needed.```

Reply:
Updated the security intro. Do I need to resubmit the merge request?

Reply:
looks like it includes the new commit. still working on my git knowledge.